test/accordian.test.js

```diff
@@ -0,0 +1,35 @@
+const aem = require('../index');
+
+test('Component Functions - Happy Path', () => {
+    let testContent = {
+        "jcr:primaryType":"nt:unstructured",
+        "sling:resourceType":"core/wcm/components/accordion/v1/accordian", 
+        "id":"abc", 
+        "expandedItems":"header",
+        "singleExpansion":false,
+        "headingElement": "h3"
+    }
+
+    let accordian = new aem.components.core.Accordian();
+
+    accordian.setExpandedItems("header");
+    accordian.setSingleExpansion(false);
+    accordian.setHeadingElement("h3");
+    accordian.setID("abc");
+
+    expect(accordian.getData()).toStrictEqual(testContent);
+<<<<<<< Updated upstream
```

Comment:
Looks like this had an issue with the stash / apply stash. Honestly not exactly sure what order of ops lead to this particular problem, but in either case solution is same: delete lines (going backwards): 34, 22, 21.